cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HBaseVersion.java

```diff
@@ -103,7 +103,7 @@ public String getMajorVersion() {
             ver.getClassifier().startsWith(CDH58_CLASSIFIER))) {
           currentVersion = Version.HBASE_12_CDH57;
         } else {
-          currentVersion = Version.UNKNOWN;
+          currentVersion = Version.HBASE_11;
```

Comment:
Please put a comment here so we know that the 1.2 code is compatible with the 1.1 module